### DISCUSSION TITLE
Can not read properties of null reading signIn

### DISCUSSION THREAD

**Discussion by ttudorandrei** [abacritt/angularx-social-login#510] [Q&A] [answered]
I'm using Angular 13 and the latest version of the package. I am able to render the login button however, when I try to use the Google Login I am getting this error in the console and I can't find anything online on how to sort this:

![image](https://user-images.githubusercontent.com/76169655/172413922-b8c9a753-552c-49fa-ad13-549e13e220f3.png)

Anybody can point me into the right direction?

**Comment by Heatmanofurioso** [comment, marked as answer]
Hi @ttudorandrei 
There is an open issue for this in https://github.com/abacritt/angularx-social-login/issues/504
And it's being addressed in this PR https://github.com/abacritt/angularx-social-login/pull/507
